src/main/resources/application.conf

```diff
@@ -4,6 +4,7 @@ akka {
   loglevel = "DEBUG"
   stdout-loglevel = "OFF"
   log-dead-letters = off
+  actor.guardian-supervisor-strategy = "config.GuardianSupervisorStrategy"
```

Comment:
I validated that this configuration was applied successfully by logging the `actorSystem.settings.SupervisorStrategyClass` value in `src/main/scala/RemoraApp.scala` while running locally using docker compose